### PR TITLE
Djwester/add attest

### PR DESCRIPTION
**Pull request by djwester**

### CODE REVIEW

File: test/components/ally-test-script.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        .catch(
+            function (err){
+                var i;
+                for (i=0; i < my_res.violations.length; i++) {
+                    console.log(JSON.stringify(my_res.violations[i]));
+                }
+                throw err
+            })
+        });
+    }
```
Comment (alogan519) [inline test/components/ally-test-script.js]: something with your brackets looks messed up, should definitely have a closing one at the far left.

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,10 @@ language: node_js
 node_js: node
 dist: trusty
 sudo: required
+before_install:
+  - printf "_auth=${AGORA_TOKEN}\nalways-auth = true\nemail = Dave.Westerveld@d2l.com" > .npmrc
```
Comment (alogan519) [inline .travis.yml]: should this be checked in?

File: test/components/ally-test-script.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+var isAttestInstalled = function(){
+    
+    try {
+        attest.reset();
+    }
+    catch (err){
+        return false
```
Comment (alogan519) [inline test/components/ally-test-script.js]: missing some semi-colons in this file I think

File: .travis.yml (hunk 2 above)
Comment (djwester) [reply .travis.yml]: You mean because it has my email?

**Comment by djwester** [comment]
@alogan519 any issues with me merging this in?

**Comment by alogan519** [comment]
go for it

**Pull request merged**